lib/puppet/provider/gpo/lgpo.rb

```diff
@@ -5,7 +5,7 @@
   commands :lgpo => 'lgpo.exe'
 
   def exists?
-    @property_hash[:ensure] == :present
```

Comment:
I think `exists?` should be kept strictly for checking `ensure => present`. We should use another method to checking if the resource is deleted. See for example what was done with `in_position?` in https://github.com/hercules-team/augeasproviders_postgresql/blob/master/lib/puppet/type/pg_hba.rb#L21